src/handshakes/abstract-handshake.ts

```diff
@@ -1,13 +1,13 @@
 import {Buffer} from "buffer";
-import { AEAD, x25519, HKDF, SHA256 } from 'bcrypto';
+import { AEAD, x25519, SHA256 } from 'bcrypto';
 
 import {bytes, bytes32, uint32} from "../@types/basic";
 import {CipherState, MessageBuffer, SymmetricState} from "../@types/handshake";
 import {getHkdf} from "../utils";
 
-export abstract class AbstractHandshake {
-  protected minNonce = 0;
+const minNonce = 0;
```

Comment:
constants are in uppercase and should probably be exporeted like:
`export const MIN_NONCE = 0;`